src/main/java/aeneas/models/Level.java

```diff
@@ -135,5 +153,38 @@ public static Level loadLevel(File file) throws IOException {
     return bullpen.pieces;
   }
 
-  public abstract LevelWidgetView makeCorrespondingView();
+  public abstract LevelWidgetView makeCorrespondingView(Model model);
+
+  public abstract RadioButton getButton();
```

Comment:
again, why put this here? We already have a way to get all the radio buttons for the editor.